contrib/systemd/README.md

```diff
@@ -5,6 +5,8 @@ author: Ben Boeckel
 date: 2015-03-22
 contributors: Abdo Roig-Maranges
 updated: 2015-03-25
+contributors: benutzer193
+updated: 2017-06-01
```

Comment:
Okay, will do. Another question before I change this: Do we want `Restart=on-failure` also for the non-oneshot service files?